tests/unit/miniapps/test_sedov.cpp

```diff
@@ -0,0 +1,3688 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#ifdef _WIN32
```

Comment:
We should discuss if we want this as a unit test here...

Reply:
It can be moved wherever, as long as the tests are done.